bazel/repository_locations.bzl

```diff
@@ -1191,12 +1191,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "compiler-rt",
         project_desc = "LLVM compiler runtime library",
         project_url = "https://compiler-rt.llvm.org",
-        version = "14.0.0",
-        sha256 = "27ab7fcfb21d108093c0be766a9ed5fe18c04e4f74f936069711a312c8ae0377",
+        version = "15.0.1",
```

Comment:
Is this right? We still on clang 14.0.0? I think we should update these together with the clang.

Reply:
`15.0.2` is out - https://github.com/llvm/llvm-project/releases/tag/llvmorg-15.0.2